src/Service/Configurations/RuntimeConfigValidator.cs

```diff
@@ -283,6 +328,120 @@ public static void ValidateEntityNamesInConfig(Dictionary<string, Entity> entity
             }
         }
 
+        /// <summary>
+        /// Helper method to get the rest path for the entity if it is correctly configured.
+        /// The rest path can only be a boolean value or a string.
```

Comment:
Im not sure if path can be a boolean value. The parent `rest` can be but not the path. What does it mean if its a boolean?

Reply:
You're correct. I checked in the draft schema as well. Will update this.

Reply:
I do see a lot of entity configuration in the config file where the rest.path is set as `true` when it should have been a string. I think that's how I got the impression that the rest.path can be a bool/string. So this now becomes a breaking change to not support boolean values for rest.path. Need to discuss this further.